packages/atomic/src/components/commerce/product-template/product-template-common.tsx

```diff
@@ -0,0 +1,185 @@
+import {ResultTemplateCondition, ResultTemplatesHelpers} from '@coveo/headless';
```

Comment:
Hum ok, I missed `common/result-templates/result-templates-common` in my refactor.

Let's go with this for the time being, but there is too much repetition as it stands between those two, we need some better generic support for `results` vs `product`.